src/lib/auth0-provider.tsx

```diff
@@ -0,0 +1,52 @@
+'use client';
+
+import React from 'react';
+import {
+  Auth0Provider,
+  useAuth0,
+} from '@auth0/auth0-react';
+
+import { Actions } from '@/app/actions';
+import { CONFIG } from '@/helpers/env';
+
+export default function Provider({
+  children,
+}: React.PropsWithChildren): JSX.Element {
+  return (
+    <Auth0Provider
+      domain={CONFIG.auth0.domain}
+      clientId={CONFIG.auth0.clientId}
+      authorizationParams={{
+        redirect_uri: (typeof window !== 'undefined' && window.location.origin) || '',
+        scope: CONFIG.auth0.scopes,
+        connection: 'maffin-gcp',
+        audience: 'https://maffin',
```

Comment:
this is **needed** or otherwise auth0 returns an opaque jwt token which can't be decoded/verified.